SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/MulticastRPCService.cpp

```diff
@@ -93,13 +104,17 @@ void MulticastRPCService::ProcessChanges()
 			{
 				ComponentUpdate(Delta.EntityId, Change.ComponentId, Change.Update);
 			}
+			for (const ComponentChange& Change : Delta.ComponentsRefreshed)
+			{
+				ComponentUpdate(Delta.EntityId, Change.ComponentId, nullptr);
```

Comment:
It doesn't look like the receiving function actually cares about the component update. We could remove the param instead of passing nullptr.